kubernetes/deployment-production.tmpl

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zoomapper-production-app
+  labels:
+    app: zoomapper-production-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: zoomapper-production-app
+  template:
+    metadata:
+      labels:
+        app: zoomapper-production-app
+    spec:
+      containers:
+        - name: zoomapper-production-app
+          image: zooniverse/zoomapper:__IMAGE_TAG__
+          resources:
+            requests:
+              memory: "50Mi"
```

Comment:
we've decided to keep memory requests and limits the same. If the app needs a limit value then then ensure it is available via the request portion when the pod is allocated to a k8s node.
```suggestion
              memory: "100Mi"
```